src/rgw/rgw_rest_s3.cc

```diff
@@ -526,7 +525,28 @@ int RGWGetObj_ObjStore_S3::send_response_data(bufferlist& bl, off_t bl_ofs,
       auto iter = bl.cbegin();
       decode(rt, iter);
 
+      auto expire_iter = attrs.find(RGW_ATTR_RESTORE_EXPIRY_DATE);
+      ceph::real_time expiration_date;
+      rgw::sal::RGWRestoreStatus restore_status;
+      attr_iter = attrs.find(RGW_ATTR_RESTORE_STATUS);
+      if (attr_iter != attrs.end()) {
+        bufferlist bl = attr_iter->second;
+        auto iter = bl.cbegin();
+        decode(restore_status, iter);
+      }
+      if (expire_iter != attrs.end()) {
```

Comment:
nit: hmm..why is expire_iter and expiration_date moved back to outside "if (Temporary)" loop? I believe it is good practice to define and use variables only in the scope where they are necessary.

Reply:
oops! didn't saw that, moved it with restore status accidentally. Fixing.